db/dcrpg/internal/blockstmts.go

```diff
@@ -50,6 +50,7 @@ const (
 		height INT4,
 		size INT4,
 		is_valid BOOLEAN,
+		is_mainchain BOOLEAN,
```

Comment:
Addresses table has new column `valid_mainchain` and Blocks table has `is_mainchain`. I think using `is_mainchain` all across would ensure consistency since the variable represent the same thing but in different tables.

Reply:
No need to address this, I now understand why you used `valid_mainchain`